brainvisa/toolboxes/tools/processes/documentation/generateDocumentation.py

```diff
@@ -103,28 +104,28 @@ def generateHTMLDocumentation(processInfoOrId, translators, context, ontology):
         if not os.path.isdir(p):
             os.makedirs(p)
         # Main page
-        f = open(htmlFileName, 'w')
-        print('<html>', file=f)
-        print('<head>', file=f)
-        print('<title>' + tr.translate(processInfo.name) + '</title>', file=f)
+        f = io.open(htmlFileName, 'w', encoding='utf-8')
+        f.write(six.text_type("""
+<html>
+<head>
+<title>{process_info_name}</title>
+<meta http-equiv="Content-Type" content="text/html; charset={default_encoding}">
+<meta content="BrainVISA {short_version}" name="generator">
+<link rel="stylesheet" href="../../axon.css" media="screen" />
+</head>
+<body>
+<h1><a href="bvshowprocess://{process_info_id}"><img src="../../images/icons/icon_process.png" border="0"></a>
+<a name="bv_process%{process_info_id}">{process_info_name}</a></h1>
+        """.format(
+            default_encoding=sys.getdefaultencoding(),
```

Comment:
The encoding is now always UTF-8

Reply:
So you think I can directly change the string to use `utf-8` for `charset?

Reply:
Yes